index.coffee

```diff
@@ -48,6 +48,9 @@ class ProgressTracker
 	addLayer: (id) ->
 		@layers[id] = { progress: null, coalesced: false }
 
+	linkLayer: (tracker, id) ->
```

Comment:
For consistency with other methods of this class that  take id as first parameter, maybe it would be nice to reverse the order of arguments for this function